Document alerts and notifications for NOM 1.6 (#106)
neo4j/docs-ops-manager: modules/ROOT/pages/monitoring/alerts-conditions.adoc

image::alerts-panel.png[width=800]

Information about Alerts is regularly updated.

== Managing Alerts

Alert details can be shown in the Alerts tab of the Alerts page of the selected DBMS.

[NOTE]
====
Only Alerts that are relevant to the selected DBMS are shown on the Alerts tab.
====

An Alert can be locally dismissed for the specified period of time.
Dismissing an Alert temporarily removes it from the Alerts table and Alerts panel in the Home view.
In the current implementation, Alert dismissal is local, meaning it is only effective for the current browser instance.
After the dismissal time the Alert will reappear in the Alerts table.

== Managing Conditions

Conditions defined in the scope of the selected DBMS can be managed in the Conditions tab of the Alert page of the selected DBMS.

image::conditions-table.png[width=800]

A Condition can be *enabled* or *disabled* by turning the switch to the left of the condition on and off.
If a Condition is not enabled, it will not result in an Alert being created.

All Conditions of the selected DBMS can be *enabled* or *disabled* at once by clicking the switch located in the table header.

> A Condition can be *deleted* using the "..." menu located at the end of every Condition table row.

Reviewer: Debatable whether we need to mention this feature since it should be pretty self-explanatory, but well written with the disclaimer below!

= Alerts and Conditions

> :description: This section describes how to manage Alerts and Conditions in Neo4j Ops Manager.

Reviewer 2: Do we need camel case here?

== Alerts on the Home page

> Home page aggregates the Alerts for your individual monitored systems.

Reviewer 2: Why does _Alerts_ need to be capitalized?

Total Alert count and alert count for the currently selected DBMS are displayed in the sidebar and DBMS dropdown.
The color of the count labels tells the maximum severity of the Alerts this count aggregates.

Nodes on the topology of the NOM installation also contain Alert counts.
Clicking on a node with Alerts displays relevant Alerts in the panel to the right of the topology.

> Location of the alert can help you understand the location of the problem, so it can be resolved faster.

Reviewer 2: Either capitalize or not, consistency is key. I vote for no caps.
```suggestion
The location of the alert can help you understand the location of the problem, so it can be resolved faster.
```

> The Alert generated for a metric threshold that has been exceeded is illustrated in the image below.

Reviewer 2: ```suggestion
The Alert generated for an exceeded metric threshold is illustrated in the image below.
```

image::alerts-panel.png[width=800]

Information about Alerts is regularly updated.

== Managing Alerts

Alert details can be shown in the Alerts tab of the Alerts page of the selected DBMS.

[NOTE]
====
Only Alerts that are relevant to the selected DBMS are shown on the Alerts tab.
====

An Alert can be locally dismissed for the specified period of time.
Dismissing an Alert temporarily removes it from the Alerts table and Alerts panel in the Home view.
In the current implementation, Alert dismissal is local, meaning it is only effective for the current browser instance.
After the dismissal time the Alert will reappear in the Alerts table.

[#_managing_conditions]
== Managing Conditions

> Conditions defined in the scope of the selected DBMS can be managed in the Conditions tab of the Alert page of the selected DBMS.

Reviewer 2: As above, why do we need capitalization?

image::conditions-table.png[width=800]

> A Condition can be *enabled* or *disabled* by turning the switch to the left of the condition on and off.

Reviewer 2: ```suggestion
A Condition can be *enabled* or *disabled* with the switch to the left of the condition.
```
We try not to be too specific in the instructions, to avoid belittling our users.

> If a Condition is not enabled, it will not result in an Alert being created.

Reviewer 2: ```suggestion
If a Condition is disabled, it will result in an Alert not being created.
```

> All Conditions of the selected DBMS can be *enabled* or *disabled* at once by clicking the switch located in the table header.

Reviewer 2: ```suggestion
*All* Conditions of the selected DBMS can be *enabled* or *disabled* at once with the switch located in the table header.
```

A Condition can be *deleted* using the "..." menu located at the end of every Condition table row.

[IMPORTANT]
====

> If an event-based Condition is deleted, and that event occurs, then a default Condition will be created for that event.

Reviewer 2: ```suggestion
If an event-based Condition is deleted, and that event occurs, then a default Condition is created for that event.
```
